### PR TITLE
New welcome page for 4.4 LTS & 5.0

### PR DESCRIPTION
**Pull request by yceruto**
| Q             | A
| ------------- | ---
| Branch?       | 4.4
| Bug fix?      | no
| New feature?  | yes
| BC breaks?    | no
| Deprecations? | no
| Tests pass?   | yes
| Fixed tickets | -
| License       | MIT
| Doc PR        | -

A new LTS version of Symfony is coming and a new major version too. That deserves a little more :heart: than usual.

This picture simulates two components of the life: the Ocean and the sky/Universe, with the aim of inspiring calm at the same time desire to build good & great things:

 welcome-page

The HUE value changes randomly each time you enter the homepage. That's just an idea to make it dynamic. I invite you to try it and send your ideas too. I'd like this experience to be extremely exciting for newcomers.

About all text & messages, this would be a good time to say what people want to hear :) Symfony's amazing \o/. Some communication skills are required here.

I'm not a designer, so I'd want some expert on this topic destroying the proposal making it even better :)

HTML+CSS only: I tried to use some modern features that were compatible with most browsers, but I might be forgetting something.

The icons were taken from https://material.io/resources/icons/ (free), but I don't know how to deal with this license:
> Our icons are free for everyone to use. Please don’t try to sell them.
> Available under [Apache license version 2.0](https://www.apache.org/licenses/LICENSE-2.0.html).

Should we copy it somewhere? should it be visible to the user?

**Before:**
old-welcome-page

A good first impression is really, really important. Is it worth it? Thoughts!

ping to my friend @javiereguiluz this is for sure something you would like to check.

Cheers!

### CODE REVIEW

File: src/Symfony/Component/HttpKernel/Resources/welcome.html.php [hunk 1]
```diff
@@ -4,64 +4,102 @@
     <meta charset="UTF-8" />
     <title>Welcome!</title>
     <style>
-        body { background: #F5F5F5; font: 18px/1.5 sans-serif; }
+        <?php $hue = random_int(0, 360); ?>
```
Comment (Simperfit) [inline src/Symfony/Component/HttpKernel/Resources/welcome.html.php]: $hue ahahah

**Comment by javiereguiluz** [comment]
I'm on a smartphone so I can't review this properly, but I want to tell that I absolutely love this proposal. Yonel, what an amazing work you did here. I couldn't like it more ♥️

**Comment by Simperfit** [comment]
I really love the proposal !

**Comment by alanpoulain** [comment]
Wouldn't it be even greater if the "sea" was slowly moving with some CSS animation? 😃

**Comment by yceruto** [comment]
> Wouldn't it be even greater if the "sea" was slowly moving with some CSS animation? smiley

Yes, that would be super nice! I was tempted since I started working on this to give a subtle animation to some of the elements, the stars or the sea or the texts, but I'd need to look for some examples. Do you know how to do it right?

**Comment by yceruto** [comment]
If we decide add some animation, it shouldn't be a distraction, perhaps a limited animation (not infinite) of some element (not all) should be enough.

**Comment by wouterj** [comment]
First of all: :heart_eyes:  :heart_eyes:  it looks amazing!

I would not add to much noise (I think the current changing hue's might be already too much, but that might have more to do with the GIF than the actual result).

**Comment by yceruto** [comment]
I've created an **inverted** version too, **what do you like more?**

| vote :tada: => initial | vote :heart: => inverted |
| --- | --- |
| welcome-dark | welcome-light |

**Comment by yceruto** [comment]
I've also removed the :star: element from the **initial** version:
 * vote :+1: to remove it.
 * vote :-1: to keep it.

**Review by ro0NL — APPROVED** [review]
less is more :ok_hand:

**Comment by Tobion** [comment]
I really like the night sky with the stars. The Symfony logo totally reminded me of the moon. It would be funny to put the SF text on an allusion of a moon and add some glow. Something like https://www.vectorstock.com/royalty-free-vector/realistic-beautiful-moon-on-transparent-vector-19727271

**Comment by Pierstoval** [comment]
I think it's the best design cleanup Symfony could ever hope for 😄

**Comment by gmponos** [comment]
I liked the star element and maybe there could be a wordplay somewhere like "Star the symfony project". Not sure if it will make it worse though...

Anyway really great result.

**Review by fabpot — APPROVED** [review]
Great work! Let's merge and iterate from there.

**Comment by fabpot** [comment]
Thank you @yceruto.

**Comment by fabpot** [comment]
@yceruto I've just tested on a brand new project and I've the following comments:

 * The directory is wrong and the vertical alignment does not seem correct

image

 * I've a too small screen which means that I see icons but not their meaning, even if I scroll down

image

**Comment by yceruto** [comment]
See https://github.com/symfony/symfony/pull/33510

**Pull request merged**